src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsModelValidator.cs

```diff
@@ -5,15 +5,21 @@
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
+using Microsoft.AspNet.Mvc.DataAnnotations;
 using Microsoft.Extensions.Localization;
 
 namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
 {
+    /// <summary>
+    /// Provides server-side validation of Models
+    /// </summary>
     public class DataAnnotationsModelValidator : IModelValidator
     {
         private IStringLocalizer _stringLocalizer;
 
-        public DataAnnotationsModelValidator(ValidationAttribute attribute, IStringLocalizer stringLocalizer)
+        public DataAnnotationsModelValidator(
+            ValidationAttribute attribute,
+            IStringLocalizer stringLocalizer)
```

Comment:
FYI the previous single line was just fine. No need to revert but this isn't a type of change worth making (unless the line is too long).